src/Swarm/Language/Syntax.hs

```diff
@@ -120,12 +120,12 @@ dirInfo d = case d of
   DLeft -> relative (\(V2 x y) -> V2 (- y) x)
   DRight -> relative (\(V2 x y) -> V2 y (- x))
   DBack -> relative (\(V2 x y) -> V2 (- x) (- y))
+  DDown -> relative (const down)
```

Comment:
I think I might have at some point advocated for `down` being a cardinal direction, but now I need it to be allowed without a compass :sweat_smile: I do not think it's worth coming up with another name for the set of directions that are cardinal but not `down`. Writing `/= down` everywhere is also an option, but not a good one. :shrug:

Reply:
Yeah, making `down` a relative direction makes sense, given the existence of `compass`.  You should not need a `compass` to find the ground. :laughing: